es-rest-client-1.3/core/src/main/java/org/elasticsearch/action/search/SearchResponseHelper.java

```diff
@@ -33,6 +35,20 @@ public SearchResponseHelper(final SearchRequest request) {
             aggregationsManifest = AggregationsManifest.fromSource(nodeMapValue(source.get("aggregations"), String.class, Object.class));
         } else if (source.containsKey("aggs")) {
             aggregationsManifest = AggregationsManifest.fromSource(nodeMapValue(source.get("aggs"), String.class, Object.class));
+        } else if (source.containsKey("aggregations_binary")) {
+            //
+            // TODO This can probably done in a more efficient way and probably not have to drag Jackson in to do it.
+            //
+            Map<String, Object> jsonMap;
+            try {
+                String aggString = new String((byte[])source.get("aggregations_binary"), "UTF-8");
+                aggString = aggString.replace("\n", " ");
+                aggString = aggString.replaceAll("\\s+", " ");
+                jsonMap = new ObjectMapper().readValue(aggString, new TypeReference<HashMap<String,Object>>(){});
```

Comment:
this is what MapFunctions#toMap does. I'll switch to using that, and then we won't need Jack anymore!